src/measure_theory/interval_integral.lean

```diff
@@ -394,6 +394,20 @@ begin
     preimage_neg_Ioc, neg_neg, restrict_congr_set Ico_ae_eq_Ioc]
 end
 
+/-- If two functions are equal in the relevant interval, their interval integrals also equal. -/
+lemma integral_congr (a b : ℝ) (f g : ℝ → E) (h : ∀ x ∈ interval a b, f x = g x) :
```

Comment:
```suggestion
lemma integral_congr {a b : ℝ} (f g : ℝ → E) (h : ∀ x ∈ interval a b, f x = g x) :
```
When arguments can be inferred from the other hypotheses, we make them implicit.  (It's clear-cut here for `a` and `b`, arguably `f` and `g` should be implicit here too but it could go either way.)